app/components/group-page/group-page.coffee

```diff
@@ -0,0 +1,13 @@
+module.exports = 
+  url: '/groups/:groupId'
+  template: require('app/components/group-page/group-page.html')
```

Comment:
should use a relative require when what you're requiring is in the same component.